### PR TITLE
🌐 Add Japanese translation for Deployment - Docker

### PR DESCRIPTION
**Pull request by tokusumi**
This PR translates deployment/docker.md in Japanese.

To all reviewers. This section is only separated from #1948 (with few changes), but please review it.

### CODE REVIEW

File: docs/ja/docs/deployment/docker.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
+# Dockerを使用したデプロイ
+
+このセクションでは以下の使い方の紹介とガイドへのリンクが確認できます:
+
+* **5分**程度で、**FastAPI** のアプリケーションを、パフォーマンスを最大限に発揮するようなDokcerイメージ (コンテナ)にする。
```
Comment (komtaki) [inline docs/ja/docs/deployment/docker.md:5]:
```suggestion
* **5分**程度で、**FastAPI** のアプリケーションを、パフォーマンスを最大限に発揮するDokcerイメージ (コンテナ)にする。
```

More simply.

**Review by komtaki — COMMENTED** [review]
Thank for your nice work.

I commented one suggestion.

**Comment by tokusumi** [comment]
Thank you for your review!

**Review by komtaki — APPROVED** [review]
Thank you : )

File: docs/ja/docs/deployment/docker.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+```Dockerfile
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.7
+
+COPY ./app /app
+```
+
+### より大きなアプリケーション
+
+[Bigger Applications with Multiple Files](tutorial/bigger-applications.md){.internal-link target=_blank} セクションを倣う場合は、`Dockerfile` は以下の様になるかもしれません:
```
Comment (sh0nk) [inline docs/ja/docs/deployment/docker.md:36]:
```suggestion
[Bigger Applications with Multiple Files](tutorial/bigger-applications.md){.internal-link target=_blank} セクションに倣う場合は、`Dockerfile` は上記の代わりに、以下の様になるかもしれません:
```
Add the implication of "instead".

File: docs/ja/docs/deployment/docker.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+```Dockerfile
+FROM tiangolo/uvicorn-gunicorn-fastapi:python3.7
+
+COPY ./app /app/app
+```
+
+### Raspberry Piなどのアーキテクチャ
+
+Raspberry Pi (ARMプロセッサ搭載)やそれ以外のアーキテクチャでDockerが作動している場合、(マルチアーキテクチャである) Pythonベースイメージを使って、一から`Dockerfile`を作成し、Uvicornを使用できます。
```
Comment (sh0nk) [inline docs/ja/docs/deployment/docker.md:46]:
```suggestion
Raspberry Pi (ARMプロセッサ搭載)やそれ以外のアーキテクチャでDockerが作動している場合、(マルチアーキテクチャである) Pythonベースイメージを使って、一から`Dockerfile`を作成し、Uvicornを単体で使用できます。
```

Add the implication of "alone".

File: docs/ja/docs/deployment/docker.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
+# Dockerを使用したデプロイ
+
+このセクションでは以下の使い方の紹介とガイドへのリンクが確認できます:
+
+* **5分**程度で、**FastAPI** のアプリケーションを、パフォーマンスを最大限に発揮するDokcerイメージ (コンテナ)にする。
```
Comment (sh0nk) [inline docs/ja/docs/deployment/docker.md:5]:
```suggestion
* **5分**程度で、**FastAPI** のアプリケーションを、パフォーマンスを最大限に発揮するDockerイメージ (コンテナ)にする。
```
Just a typo.

File: docs/ja/docs/deployment/docker.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+* (オプション) 開発者として必要な範囲でHTTPSを理解する。
+* **20分**程度で、自動的なHTTPS生成とともにDockerのSwarmモード クラスタをセットアップする (月5ドルのシンプルなサーバー上で)。
+* **10分**程度で、DockerのSwarmモード クラスタを使って、HTTPSなどを使用した完全な**FastAPI** アプリケーションの作成とデプロイ。
+
+デプロイのために、<a href="https://www.docker.com/" class="external-link" target="_blank">**Docker**</a> を利用できます。セキュリティ、再現性、開発のシンプルさなどに利点があります。
+
+Dockerを使う場合、公式のDokcerイメージが利用できます:
```
Comment (sh0nk) [inline docs/ja/docs/deployment/docker.md:12]:
```suggestion
Dockerを使う場合、公式のDockerイメージが利用できます:
```
Just a typo.

File: docs/ja/docs/deployment/docker.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+def read_root():
+    return {"Hello": "World"}
+
+
+@app.get("/items/{item_id}")
+def read_item(item_id: int, q: Optional[str] = None):
+    return {"item_id": item_id, "q": q}
+```
+
+* ここでは、以下の様なディレクトリ構造になっているべきです:
```
Comment (sh0nk) [inline docs/ja/docs/deployment/docker.md:85]:
```suggestion
* ここでは、以下の様なディレクトリ構造になっているはずです:
```

File: docs/ja/docs/deployment/docker.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+
+自動生成された対話的APIドキュメントが確認できます (<a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a>によって提供される):
+
+![Swagger UI](https://fastapi.tiangolo.com/img/index/index-01-swagger-ui-simple.png)
+
+### その他のAPIドキュメント
+
+また同様に、<a href="http://192.168.99.100/redoc" class="external-link" target="_blank">http://192.168.99.100/redoc</a> や <a href="http://127.0.0.1/redoc" class="external-link" target="_blank">http://127.0.0.1/redoc</a> (もしくはDockerホストを使用したこれらと同等のもの) を開いて下さい。
+
+他の自動生成された対話的なAPIドキュメントが確認できます (<a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>によって提供される):
```
Comment (sh0nk) [inline docs/ja/docs/deployment/docker.md:145]:
```suggestion
他の自動生成された対話的なAPIドキュメントが確認できます (<a href="https://github.com/Rebilly/ReDoc" class="external-link" target="_blank">ReDoc</a>によって提供されます):
```

File: docs/ja/docs/deployment/docker.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+```JSON
+{"item_id": 5, "q": "somequery"}
+```
+
+### 対話的APIドキュメント
+
+ここで、<a href="http://192.168.99.100/docs" class="external-link" target="_blank">http://192.168.99.100/docs</a> や <a href="http://127.0.0.1/docs" class="external-link" target="_blank">http://127.0.0.1/docs</a> (もしくはDockerホストを使用したこれらと同等のもの) を開いて下さい。
+
+自動生成された対話的APIドキュメントが確認できます (<a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a>によって提供される):
```
Comment (sh0nk) [inline docs/ja/docs/deployment/docker.md:137]:
```suggestion
自動生成された対話的APIドキュメントが確認できます (<a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a>によって提供されます):
```

File: docs/ja/docs/deployment/docker.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+
+DockerコンテナのURLで確認できるはずです。例えば: <a href="http://192.168.99.100/items/5?q=somequery" class="external-link" target="_blank">http://192.168.99.100/items/5?q=somequery</a> や <a href="http://127.0.0.1/items/5?q=somequery" class="external-link" target="_blank">http://127.0.0.1/items/5?q=somequery</a> (もしくはDockerホストを使用したこれらと同等のもの)。
+
+以下の様なものが返されます:
+
+```JSON
+{"item_id": 5, "q": "somequery"}
+```
+
+### 対話的APIドキュメント
```
Comment (sh0nk) [inline docs/ja/docs/deployment/docker.md:133]:
```suggestion
## 対話的APIドキュメント
```

File: docs/ja/docs/deployment/docker.md [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+
+ここで、<a href="http://192.168.99.100/docs" class="external-link" target="_blank">http://192.168.99.100/docs</a> や <a href="http://127.0.0.1/docs" class="external-link" target="_blank">http://127.0.0.1/docs</a> (もしくはDockerホストを使用したこれらと同等のもの) を開いて下さい。
+
+自動生成された対話的APIドキュメントが確認できます (<a href="https://github.com/swagger-api/swagger-ui" class="external-link" target="_blank">Swagger UI</a>によって提供される):
+
+![Swagger UI](https://fastapi.tiangolo.com/img/index/index-01-swagger-ui-simple.png)
+
+### その他のAPIドキュメント
```
Comment (sh0nk) [inline docs/ja/docs/deployment/docker.md:141]:
```suggestion
## その他のAPIドキュメント
```

**Review by sh0nk — COMMENTED** [review]
This is my first time review on PRs. If I don't follow any convention etc., please kindly forget about the comments!

**Comment by tokusumi** [comment]
Thank you for your first review! @sh0nk
I applied your suggestions. Please check it again.

**Review by sh0nk — APPROVED** [review]
Thanks for the fix. LGTM 👍

**Comment by tiangolo** [comment]
Awesome @tokusumi ! :rocket: :cake: 

And thanks for the reviews @sh0nk and @komtaki ! :bow: :cake: :coffee:

**Pull request merged**